src/git.ts

```diff
@@ -48,7 +64,11 @@ export async function deploy(action: ActionInterface): Promise<Status> {
     const commitMessage = !isNullOrUndefined(action.commitMessage)
       ? (action.commitMessage as string)
       : `Deploying to ${action.branch}${
-          process.env.GITHUB_SHA ? ` from @ ${process.env.GITHUB_SHA}` : ''
+          process.env.GITHUB_SHA
+            ? ` from @ ${action.repositoryName ? action.repositoryName : ''}${
```

Comment:
I think you want the @ inbetween:
```suggestion
            ? ` from ${action.repositoryName ? action.repositoryName : ''}@${
```

Reply:
I'm testing a change to this now, I wanna see if `from @ JamesIves/repo@123` is also valid as that way we keep the commit the messages the same.

Reply:
Appears to have worked 🎉

Reply:
Just kidding, this needs to link the commit from the original repo. I've updated this once again so it links `${process.env.GITHUB_REPOSITORY}@${process.env.GITHUB_SHA}`